pokedex/db/load.py

```diff
@@ -193,7 +193,39 @@ def load(session, tables=[], directory=None, drop_tables=False, verbose=False, s
 
     print_start('Creating tables')
     for n, table in enumerate(table_objs):
-        table.create(bind=engine)
+        try:
+            table.create(bind=engine)
+        
+        # This except is for SQLite
+        except sqlalchemy.exc.OperationalError as error:
+            # All the databases create extra spaces or new lines so this is the best way I can see to handle it
+            if "table {} already exists".format(table) in error.orig.__str__():
```

Comment:
Good point I don't know why I used__str__() I'll work on this.
As for why use error.orig, error by itself outputs this
```
(sqlite3.OperationalError) table language
s already exist s [SQL: '\nCREATE TABLE languages (\n\tid INTEGER NOT NULL, \n\tiso639 VARCHAR(79) NOT NULL, \n
\tiso3166 VARCHAR(79) NOT NULL, \n\tidentifier VARCHAR(79) NOT NULL, \n\tofficial BOOLEAN NOT NULL, \n\t"order
" INTEGER, \n\tPRIMARY KEY (id), \n\tCHECK (official IN (0, 1))\n)\n\n'] (Background on this error at: http://
sqlalche.me/e/e3q8)
```
and error.orig outputs
```
table languages already exists
```
I could just use `error` but it creates unneeded spaces that would need to be handled too.
---
I agree I don't like the repetition but I just couldn't think of other ways to make it look nicer without making it into a separate function and a bit harder to follow.
I'll try a few things out and see what I can think of to make it more neat.

Reply:
Ah, thanks for the explanation.

Sounds good. Let me know when it's ready.

Reply:
I did 3 commits since I forgot to add an exception on the first one and the second one had an indentation error.
Those commits should implement everything needed.

d8a2c176b51735361c402c1886e33cfb789c81cc
149e07a957d07ac6a09af9f827baed7bf69835b3
ff87860d6a6a6a3baf37e5e5f4dd047d98104288